common/changes/@itwin/delete-imodel-react/vp-Update-delete-imodel-package_2025-04-23-12-16.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/delete-imodel-react",
+      "comment": "Updated imodel-delete to itwin-react V3, removed support from react 17.x",
+      "type": "major"
```

Comment:
hm..
maybe we could do major and actually support only version 18 and maybe 19 of react (that needs testing though).
Also, iTiwnUI v3 requires css import on consumer side kkind of..